### DISCUSSION TITLE
Is Actix Has Smtp Client?

### DISCUSSION THREAD

**Discussion by Necoo33** [actix/actix-extras#343] [Q&A] [answered]
Hi, i'm planning to use actix-web for my project and i want to send emails to individuals. Is actix-web has any smtp client or is compatible to any other rust email client?

**Comment by robjtede** [comment, marked as answer]
Any SMTP client you can find on crates.io will work.

If it's pool-based, then you will be able construct the client/pool once and put it in app data.
